### PR TITLE
Turns off autocomplete on some backend forms

### PR DESCRIPTION
**Pull request by clarissalimab**
Closes [tix #3030](https://github.com/CommitChange/tix/issues/3030).

### CODE REVIEW

File: app/views/billing_subscriptions/_new_modal.html.erb [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-		<% end %>
+  <div class='modal-body'>
+    <% if current_role?(:super_admin) && @nonprofit %>
+    <p><strong>Super admin mode</strong></p>
+    <p><%= "Selected nonprofit: #{@nonprofit.name} (id #{@nonprofit.id})" %></p>
+    <% end %>
 
-		<form class='cardForm' parsley-validate>
-			<!--= on 'submit' create_billing_sub -->
+    <form autocomplete='off' class='cardForm' parsley-validate>
```
Comment (wwahammy) [inline app/views/billing_subscriptions/_new_modal.html.erb]: The autocomplete shouldn't be turned off on this specific form. It's used for payment by the nonprofit.

File: app/views/billing_subscriptions/_new_modal.html.erb [hunk 2]
```diff
@@ -1,35 +1,37 @@
 <%- # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later -%>
 <div class='modal skinny' id='newBillingSubscriptionModal'>
-	<%= render 'common/modal_header', title: "Movin' on up!" %>
```
Comment (wwahammy) [inline app/views/billing_subscriptions/_new_modal.html.erb]: Could you undo all of the formatting changes here? We don't have formal formatting rules on this repo and it makes it unclear what actually changed.

**Review by wwahammy — CHANGES_REQUESTED** [review]
Thanks for the good work @clarissalimab. Could you resubmit without the formatting changes? It makes the PR really hard to review and I want to make sure I know what exactly changed.

File: app/views/billing_subscriptions/_new_modal.html.erb (hunk 2 above)
Comment (clarissalimab) [reply app/views/billing_subscriptions/_new_modal.html.erb]: Okay!

**Comment by wwahammy** [comment]
The form in `app/views/nonprofits/supporters/_new_modal.html.erb` needs to have `autocomplete='off'` added.

**Review by wwahammy — CHANGES_REQUESTED** [review]
It all looks good to me. Only change needed is for the form in `app/views/nonprofits/supporters/_new_modal.html.erb` needs to have `autocomplete='off'` added.

**Review by wwahammy — APPROVED** [review]

**Pull request merged**